requirements/dev.txt

```diff
@@ -3,4 +3,6 @@ hupper>=1.9
 pip-tools>=1.0
 pyramid_debugtoolbar>=2.5
 pip-api
-repository-service-tuf
```

Comment:
It is now solved with the new releases of RSTUF that support tuf 4.0.0 :)

Reply:
The new release has another conflict with `warehouse`. See: https://github.com/pypi/warehouse/pull/15958#issuecomment-2131200368